### ISSUE TITLE
TypeError: Cannot read property 'optionalFeatures' of undefined

### ISSUE THREAD

**Issue by hansagames** [MozillaReality/WebXR-emulator-extension#268]
Hi,
chrome 88 again, after upgrading to latest fix, issue with context is solved but now get such error:

```
TypeError: Cannot read property 'optionalFeatures' of undefined
```
anyone else have it, as far as I got looks like its permission stuff,

will investigate further.

**Comment by hansagames** [comment]
Happens on `navigator.xr.requestSession("immersive-vr")`

**Comment by takahirox** [comment]
Hi, thanks for the report. I couldn't reproduce it. Do you have any demos where you encountered the problem?

**Comment by hansagames** [comment]
https://immersive-web.github.io/webxr-samples/immersive-vr-session.html
this for example triggers it for me

**Comment by takahirox** [comment]
Thanks, I reproduced the problem. Let me look into...

**Comment by takahirox** [comment]
Figured out the root issue and I will fix it soon.

**Comment by takahirox** [comment]
I have fixed the problem in #270. I will urgently release the new revision. Until it will be released on web stores, please manually download the latest revision from the repository and install.

https://github.com/MozillaReality/WebXR-emulator-extension#how-to-install-the-newest-version

Thanks for the bug report!

**Comment by takahirox** [comment]
The new revision has been released at the Firefox/Chrome web stores.
